scripts/test_e2e.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# test_e2e.sh
+# Execute end-to-end (e2e) tests to verify that everything is working right
+# from the end user perpsective
+set -xeuo pipefail
+
+REPO_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )/.."
+
+# Use a version of Go that supports Go Modules
+export GO111MODULES=on
+GOMOD_CACHE=$(go env GOPATH)/src/mod
```

Comment:
@carolynvs 
In Go1.11beta the location of the module cache is `$GOPATH/src/mod`
However on tip the location has changed to `$GOPATH/pkg/mod`[1]

Maybe you could just declare;
```bash
clearGoModCache () {
  # The sudo is a necessary workaround until go is fixed
  sudo rm -fr $GOPATH/*
}
```

**ref:**
1. https://go-review.googlesource.com/c/go/+/126755

Reply:
I think this should be ignored until the change is in an official Go release as we do not work with go from tip.